includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -23,12 +23,24 @@ public function setHrefs( $hrefs ) {
 	/**
 	 * @param $name string key for href
 	 * @param $lang string two letter language code
+	 * @param [$variables] array key-value variables that will be replaced in the href
 	 * @return string full URL, in case of lang specific URL missing, default one is returned
 	 */
-	public function getHref( $name, $lang ) {
+	public function getHref( $name, $lang, $variables = null ) {
 		$lang = $this->getLangWithFallback( $lang );
 
-		return $this->hrefs[ $lang ][ $name ] ?? $this->hrefs[ 'default' ][ $name ];
+		$href = $this->hrefs[ $lang ][ $name ] ?? $this->hrefs[ 'default' ][ $name ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "default" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)